nfc/libs/NfcCoreLib/lib/LibNfc/phLibNfc_Hci.c

```diff
@@ -1148,7 +1148,16 @@ NFCSTATUS phLibNfc_HciChildDevInitComplete(void* pContext, NFCSTATUS status, voi
             else
             {
                 PH_LOG_LIBNFC_INFO_STR("NFCEE initialization failed");
-                pLibCtx->tSeInfo.bSeState[bIndex] = phLibNfc_SeStateInvalid;
+
+                /* Following NCI 2.0 Figure 25: NFCEE State Transitions,
+                   on NFCEE_TRANSMISSION_ERROR, the NFCEE is to be considered unresponsive.
+                   We can remove it from the list. */
+                if (status == PH_NCINFC_STATUS_NFCEE_TRANSMISSION_ERROR)
+                {
+                    pLibCtx->tSeInfo.bSeState[bIndex] = phLibNfc_SeStateInvalid;
+                    pLibCtx->tSeInfo.tSeList[bIndex].hSecureElement = NULL;
+                    pLibCtx->tSeInfo.bSeCount--;
+                }
```

Comment:
If the error code is NFCEE_TIMEOUT_ERROR, it'll be totally ignored. Is it the expected behavior you wanted to implement.
This change will affect NCI1x behavior as well